src/components/icon-tile/types.ts

```diff
@@ -9,7 +9,7 @@ export interface IconTileProps {
   /** Pass a single child, which should be a Flight icon. For 'small' and 'medium' size, pass the 16px icon size; for other sizes pass the 24px icon size. Note that non-"color" icons will be colored using the "brandColor". */
   children: React.ReactNode
   /** Note: the "extra-large" option is not documented in the design system. It's being used for the IconTileLogo component, as used on the /{product} view pages. */
-  size: 'small' | 'medium' | 'large' | 'extra-large'
+  size?: 'small' | 'medium' | 'large' | 'extra-large'
```

Comment:
This was being treated as optional within the component, a default of 'medium' is set.